.gitignore

```diff
@@ -2,5 +2,4 @@ _build
 *.install
 *.merlin
 _opam
-.vscode
```

Comment:
Suggestion:

```
.vscode/*
!.vscode/tasks.json
```

to avoid needing to manually keep `.vscode/.gitignore` up-to-date with all files that might be in `.vscode`.